### ISSUE TITLE
[Button] Missing accessible name for icon-only button

### ISSUE THREAD

**Issue by coseeian** [processing/p5.js-website#874]
### Title

[Button] Missing accessible name for icon-only button

### Description

The icon-only button on the page is missing any accessible name, so screen readers announce only its role (e.g., “button”) without any descriptive label. As a result, users will not be able to identify the purpose or function of the button.

#### page 1
1. go to /
2.  Locate to the following element:
#hideBanner (CSS Selector)
(This issue is since to be a global issue)

#### page 2
1. go to /reference/
2.  Locate to the following element:
#main-content > div.mx-0 > astro-island > div > div.h-0.overflow-visible > div > div > button
(This issue is since to be a global issue)

### Actual Behavior

* When tabbing to the button, a screen reader announces only “button” without any label or description.
* There is no visible text or tooltip associated with the button.

### Expected Behavior

* The button should expose an accessibility name
* When tabbing to the button, A screen reader should announce a meaningful labe.

### Suggested Fix

Add an accessibility name to the button, for example:
```
<button aria-label="action">
  <svg …></svg>
</button>
```

Alternatively, include visually hidden text inside the button:
```
<button>
  <span class="visually-hidden">action</span>
  <svg …></svg>
</button>
```

### Reference

WCAG:[4.1.2 Name, Role, Value](https://www.w3.org/TR/WCAG22/#name-role-value)
WCAG-T:[F68: Failure of Success Criterion 4.1.2 due to a user interface control not having a programmatically determined name](https://www.w3.org/WAI/WCAG22/Techniques/failures/F68)
APG:[Providing Accessible Names and Descriptions](https://www.w3.org/WAI/ARIA/apg/practices/names-and-descriptions/)

### What is your operating system?

None

**Comment by eslteacher902010** [comment]
I just made a pull request for some of these things. Hope you like it. I think there are probably more buttons that could be altered but wanted to see what you thought.

**Comment by ksen0** [comment]
The fix from above is now merged on both branches: https://github.com/processing/p5.js-website/pull/911 and https://github.com/processing/p5.js-website/pull/910 - but please feel free to reopen @coseeian if needed
